thirdeye-server/src/main/java/ai/startree/thirdeye/ThirdEyeServerModule.java

```diff
@@ -69,4 +70,11 @@ public OAuthConfiguration getOAuthConfig(
       AuthConfiguration authConfiguration) {
     return authConfiguration.getOAuthConfig();
   }
+
+  @Singleton
+  @Provides
+  public BasicAuthConfiguration getBasicAuthConfig(
```

Comment:
as for reference: this is a provider. we should use this for building auth classes.